src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -391,12 +391,14 @@ def setupProcessingTask(self, procTask, taskType, inputDataset=None, inputStep=N
         procTaskCmsswHelper.setOverrideCatalog(self.overrideCatalog)
         procTaskCmsswHelper.setErrorDestinationStep(stepName=procTaskLogArch.name())
 
-        if forceMerged:
+        if forceMerged and isinstance(forceMerged, bool):
             procTaskStageHelper.setMinMergeSize(0, 0)
-        elif forceUnmerged:
+        elif forceUnmerged and isinstance(forceUnmerged, bool):
             procTaskStageHelper.disableStraightToMerge()
         else:
             procTaskStageHelper.setMinMergeSize(self.minMergeSize, self.maxMergeEvents)
+            if forceUnmerged and isinstance(forceUnmerged, list):
```

Comment:
The reason for this is that ALCARECO output isn't valid on it's own, it's just some intermediate data that is the input for another processing step. Hence under no circumstance can we allow it to be stage out straight to merge. But we also don't want to force all the PromptReco output to be written unmerged, since that causes huge inefficiencies for RECO output.